plugins/SitesManager/angularjs/sites-manager/sites-manager-site.controller.js

```diff
@@ -187,6 +187,16 @@
         };
 
         var deleteSite = function() {
+            var redirectParams = $scope.redirectParams;
+
+            // if the current idSite in the URL is the site we're deleting, then we have to make to change it. otherwise,
+            // if a user goes to another page, the invalid idSite may cause a fatal error.
+            if (broadcast.getValueFromUrl('idSite') === $scope.site.idsite) {
+                var site = $scope.adminSites.sites.find(function (site) { return site.idsite !== $scope.site.idsite });
```

Comment:
I think `.find` might not be available eg in IE10

Reply:
👍 will fix

Would be great to use something like babel-polyfill in the future :)